tests/telemetry/test_writer.py

```diff
@@ -296,12 +296,13 @@ def test_app_started_event_configuration_override(test_agent_session, run_python
 
     # DD_TRACE_AGENT_URL in gitlab is different from CI, to keep things simple we will
     # skip validating this config
-    configurations = test_agent_session.get_configurations(ignores=["DD_TRACE_AGENT_URL"])
+    configurations = test_agent_session.get_configurations(
+        ignores=["DD_TRACE_AGENT_URL", "DD_AGENT_PORT", "DD_TRACE_AGENT_PORT"], remove_seq_id=True, effective=True
+    )
```

Comment:
We should expect seq_id in the payloads in the test below I believe

Reply:
`seq_id` is tested in another test including it here will make this test very brittle. A small refactor in another file can result in a complete reordering of configs and the sequence ids in this file will shift